### ISSUE TITLE
Parent scope not accessible in controller within window

### ISSUE THREAD

**Issue by yashgt** [angular-ui/angular-google-maps#605]
I have:
&lt;body ng-controller="MyController"&gt;
    &lt;div id="map" class="google-map"&gt;
        &lt;window ....&gt;
In the window I display a &lt;form ng-controller="MyFormController"&gt;.
In the form, I can show models assigned to $scope of MyController. But in MyFormController, I am not able to access $scope models of MyController. According to controller hierarchy, this should be possible. 

Basically Looks like a controller in the window is not able to access $scope of the parent controller.
